website/pages/docs/upgrading/upgrade-to-1.5.0.mdx

```diff
@@ -0,0 +1,25 @@
+---
+layout: docs
+page_title: Upgrading to Vault 1.5.0 - Guides
+sidebar_title: Upgrade to 1.5.0
+description: |-
+  This page contains the list of deprecations and important or breaking changes
+  for Vault 1.5.0. Please read it carefully.
+---
+
+# Overview
+
+This page contains the list of deprecations and important or breaking changes
+for Vault 1.4.x compared to 1.5.0. Please read it carefully.
+
+## Raft Configuration
+
+A new Raft configuration value, `max_entry_size`, has been introduced. This value
+limits the size in bytes for a Raft entry. It applies to both Put operations and
```

Comment:
If this suggestion is correct I think it would be helpful to clarify.

As far as Put vs put, I think we should be consistent with the capitalization, but I don't much care which we use.

```suggestion
limits the size in bytes for a Raft K/V entry. It applies to both put operations and
```